Adding documentation for FB-1893.

vkrishnanfb opened the pull request:
Also a minor correction related to the deprecated sql endpoint flag.

ljsinclair commented on docs/sql-guide/functions/function-datetimeadd.md:
```diff
@@ -0,0 +1,105 @@
+
+| Argument | Data type | Description | Required | Further information |
+|---|---|---|---|---|
+| timeunit | string | String value that specifies the time unit for the duration to be added to the target date-time. | Yes | [Time units](#additional-information).|
```
The link under "further information" for timestamp should be the anchor for "timeunit" not the parent heading

ljsinclair commented on docs/sql-guide/functions/function-datetimeadd.md:
```diff
@@ -0,0 +1,105 @@
+## Examples
+
+### Add 62 years to date of birth to derive retirement date. 
+Here DATETIMEADD() is used to do a popular date arithmetic.
```
You can probably leave this out because your heading has explained what's going on.
Ultimately "popular" is a subjective viewpoint, and not guaranteed to be everyone's point of view :)

ljsinclair commented on docs/sql-guide/functions/function-datetimeadd.md:
````diff
@@ -0,0 +1,105 @@
+```
+
+### Add 24 hours wait period to last run date to derive the next run time.
+Adding 24 hours can be coded in multiple possible ways as shown below:
````
This information should be moved to a heading beneath "Additional information" because it's not just relevant to this example.

You'd do it something like this:

```
### Timeunit conversion table

| Timeunit | seconds | minutes | hours | days |
|---|---|---|---|
| day | 86400 | 1440 | 24 | 1`|
| hour | 3600 | 60 | 1 |  24 | 
| minute | 60 | 1 |   |   |
| second |  1 | 60 | 3600 | 86400  |
```

ljsinclair commented on docs/sql-guide/functions/function-datetimeadd.md:
````diff
@@ -0,0 +1,105 @@
+   3 | 2023-03-03 21:30:00 +0000 UTC | 2023-03-04 21:30:00 +0000 UTC
+   4 | 2023-03-04 21:30:00 +0000 UTC | 2023-03-05 21:30:00 +0000 UTC
+```
+### Use negative time duration to calculate date time values pointing in the past.
````
Headings aren't sentences and don't need a full-stop/period on the end.

ljsinclair commented on docs/sql-guide/sql-guide-home.md:
```diff
@@ -18,7 +18,7 @@ FeatureBase supports different types of queries that are discussed further in th
 
 {: .note}
-FeatureBase Community can run SQL through the FeatureBase CLI by [Enabling the SQL endpoint at startup](/docs/community/com-startup-connect)
+FeatureBase Community can run SQL through the commandline interactive tool [fbsql](/docs/community/com-startup-connect) or through the lattice Web UI.
```
This sentence has grammatical errors:
* "FeatureBase Community can run SQL" > FeatureBase Community runs SQL at the back-end.  Only users can use the FBSQL tool
* commandline -- this should be two words
* "interactive tool" is what's called a "tautology" which means you're saying the same thing twice -- in this case, a tool is implied to be interactive by its very existence. Also "interactive" is over-used and ultimately meaningless in all but a very few use cases. So don't use it :)
* You have two ways to run SQL here, both of which should be hyperlinks. And in fact there are 3 ways -- the query tool, the query builder, the **other** command-line SQL tool, and fbsql
* I've yet to get a clear explanation of what the "lattice Web UI" actually is. If you're talking about the Community GUI, it's best to just say that. 

An alternative approach: 

```
## Before you begin
{% include /cloud/cloud-db-create-before-begin.md %}

* [Learn how to query FeatureBase databases using terminal commands](/docs/tools/tool-fbsql)
* [Learn how to query FeatureBase databsaes using the SQL Command Line Interface](/docs/community/com-config/com-config-sql-cli-enable) <-- filename depends on existing PR "APPROVE ME FIRST! > Formatting fixes
* Login to FeatureBase Community and run queries using the Query editor or Query builder

```

vkrishnanfb commented:
Addressed all review feedback. Please note that in [docs/sql-guide/sql-guide-home.md] I added the suggested text and links to the "before you begin section". I am not familiar with the context on this proposed change, and the links are possibly broken too. My intend for the original change in this file was to remove text that was referring to a deprecated flag, that is done, beyond that I am not sure of the correctness of the new text and links.

gthrone25 reviewed (approved):
@vkrishnanfb looks good to me! On the broken link, yes that's true but I think @ljsinclair is getting ahead of another PR for fbsql that is coming in hopefully today. That will address this, but we'll own it if not! Merging in!
